src/components/ValueCard/ValueCard.story.jsx

```diff
@@ -32,6 +32,45 @@ storiesOf('Watson IoT/ValueCard', module)
       </div>
     );
   })
+  .add(
+    'small / with variables',
+    () => {
+      const size = select('size', Object.keys(CARD_SIZES), CARD_SIZES.SMALL);
+      return (
+        <div style={{ width: text('cardWidth', `${getCardMinSize('lg', size).x}px`), margin: 20 }}>
+          <ValueCard
+            title={text('title', 'Title variable is {not-working}')}
+            id="facilitycard"
+            cardVariables={object('variables', {
+              'not-working': 'working',
```

Comment:
{'not-working': (key, card)=>'working'},   variables: PropTypes.objectOf(PropTypes.oneOfType([PropTypes.string,PropTypes.func])).  typeof value === 'string' ? {value} : typeof value === 'function' ? value(key, card) : undefined